spring-expression/src/test/java/org/springframework/expression/spel/support/ReflectionHelperTests.java

```diff
@@ -364,6 +364,19 @@ public void testOptimalReflectivePropertyAccessor() throws Exception {
 				field.write(ctx, tester, "field", null));
 	}
 
+	@Test
+	void testReflectiveMethodResolver() throws AccessException {
+		MethodResolver resolver=new ReflectiveMethodResolver();
+		StandardEvaluationContext evaluationContext = new StandardEvaluationContext();
+		Object obj= Proxy.newProxyInstance(this.getClass().getClassLoader(), new Class<?>[]{Runnable.class}, new InvocationHandler() {
+			@Override
+			public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+				return null;
+			}
+		});
+		MethodExecutor mexec=resolver.resolve(evaluationContext,obj,"toString",new ArrayList<>());
+		Assert.notNull(mexec,"MethodExecutor should not be empty.");
```

Comment:
We use AssertJ for test assertions, not Spring's `org.springframework.util.Assert` class.

In any case, I'll make that change when merging the PR.